### ISSUE TITLE
ts=2018-06-27T05:47:26.148843166Z caller=images.go:21 component=sync-loop error="getting unlocked automated services: git repo not ready"

### ISSUE THREAD

**Issue by JoeyX-u** [weaveworks/flux#1173]
If the git repo  is empty before starting the flux, it will always show that err="git repo not ready, even if you commint the yaml to the directory, it still shows this error

**Comment by squaremo** [comment]
Does this look the same as #904?

Either way, yes that's a problem. I think it occurs because we try to tag HEAD as a check that we can write to the upstream repo, but if there's no commits, there's no HEAD. We may have to have a special case, when there's no commits.

**Comment by squaremo** [comment]
I'm going to close this as a duplicate of #904, but add a note there to come back and check that any fix there also covers this.
